src/hooks/useUniswapPools.js

```diff
@@ -371,6 +371,11 @@ export default function useUniswapPools(sortField, sortDirection) {
     [charts]
   );
 
+  let is30DayEnabled = false;
```

Comment:
any reason why this can't be:
`const is30DayEnabled = ethereumPriceOneMonthAgo > 0;`